test/test_torch.py

```diff
@@ -19259,6 +19259,8 @@ def inner(self, device, dtype):
 # - cpu_dtype_list (=[]), a list of torch dtypes to test the op(s) on cpu
 # - make_inplace_variant (=True), if true the inplace version of the op (op_) is also tested
 # - decorators (=[]), a list of decorators to apply to the test
+# - self_position (=-1), the position of self in the arg list, -1 means skip function check
+# - test_out (=False), whether to test the out= version of the operator
```

Comment:
Currently only support addmm and addmv, more are coming in future PR

Reply:
We're planning on killing this test suite soonish and refactoring the existing tests using OpInfos (you can see test_unary_ufunc already has copies of al the previous tests) so I wouldn't plan further improvements here.

Reply:
(This PR, however, seems fine.)